src/git-lfs/install.sh

```diff
@@ -62,29 +62,33 @@ find_version_from_git_tags() {
 
 # Get the list of GPG key servers that are reachable
 get_gpg_key_servers() {
```

Comment:
Can we bump patch version for the Feature?

Reply:
made changes